src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -69,40 +81,24 @@ public ReadsSparkSource(JavaSparkContext ctx) {
      *                  use the default split size (determined by the Hadoop input format, typically the size of one HDFS block).
      * @return RDD of (SAMRecord-backed) GATKReads from the file.
      */
-    public JavaRDD<GATKRead> getParallelReads(final String readFileName, final String referencePath, final List<SimpleInterval> intervals, long splitSize) {
-        final Configuration conf = new Configuration();
+    public JavaRDD<GATKRead> getParallelReads(final String readFileName, final String referencePath, final List<SimpleInterval> intervals, final long splitSize) {
+        final Configuration conf = ctx.hadoopConfiguration();
```

Comment:
Right, but I think we (at least *I*) don't. I'm reluctant to change it if its not necessary - I'm unclear on how it affects the splitsize code below. Also see [this exchange](https://broad.hipchat.com/history/room/1020897/2016/01/20?p=1&q=configuration&t=rid-1020897#8d8de87d-4c43-4e80-b588-8467017e1935) from a few weeks back. I don't understand how the newHadoopAPI method uses the two configs it has access to (one attached to the context  object and one passed explicitly as an arg).